Validation/MtdValidation/plugins/EtlDigiHitsValidation.cc

```diff
@@ -181,9 +205,11 @@ void EtlDigiHitsValidation::analyze(const edm::Event& iEvent, const edm::EventSe
 
     n_digi_etl[idet]++;
     size_t ncount(0);
-    ndigiPerLGAD_[idet].emplace(detId.rawId(), ncount);
-    ndigiPerLGAD_[idet].at(detId.rawId())++;
 
+    std::pair<uint8_t, uint8_t> pixel = geomUtil.pixelInModule(detId, local_point);
+    ETLPixelId pixelId(detId.rawId(), pixel.first, pixel.second);
+    ndigiPerLGAD_[idet].emplace(pixelId, ncount);
+    ndigiPerLGAD_[idet].at(pixelId)++;
```

Comment:
@ctarricone the update of this class is not needed, because of the meaning of this counter: number of DIGIs per LGAD. Each pixel has its own separate DIGI, and there is one DIGI per pixel by construction, see https://github.com/cms-sw/cmssw/blob/master/SimFastTiming/FastTimingCommon/src/ETLDeviceSim.cc#L130
and the definition of ```MTDCellId```. 

Counting DIGIs per LGAD means counting how many pixels have fired in an LGAD for a given event, and this is what we are interested to monitor here. In the case of your class, you might see just either zero or one.